community/server/src/main/java/org/neo4j/server/database/LifecycleManagingDatabase.java

```diff
@@ -114,4 +118,18 @@ public boolean isRunning()
     {
         return isRunning;
     }
+
+    private void preLoadCypherCompiler()
+    {
+        // Execute a single Cypher query to pre-load the compiler to make the first user-query snappy
+        //noinspection EmptyTryBlock
+        try (Result ignored = this.graph.execute("MATCH (a:` WEIRD LABEL NO-ONE SHOULD USE. EVER `) RETURN a"))
```

Comment:
This way, the planner will ask the kernel for stats and whatnot. I wanted to preload and JIT-compile stuff as much as possible, and this seemed like a good way of doing it.